certified-connectors/Repfabric Lead Loader/apiDefinition.swagger.json

```diff
@@ -0,0 +1,357 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Lead Loader",
+    "description": "This Connector helps automate the process of creating a \"Lead\" for Repfabric's backend system. It takes data from a csv and invokes multiple API calls that can create or get the items needed to automate the Lead creation process.",
+    "contact": {
+      "name": "Andre Roberts",
```

Comment:
@andreroberts1119 usually contact name for certified connectors is a name of support team, can we change this one to something like `Phiquest Team`? This way it won't be so personalized. Names are usually used for independent publishers.

Reply:
Understandable, changed the file.